src/controllers/queryRunner.ts

```diff
@@ -679,6 +660,74 @@ export default class QueryRunner {
         }
         await p;
 
+        this.constructCopyString(
+            copyString,
+            rowIdToRowMap,
+            rowIdToSelectionMap,
+        );
+
+        await this.writeStringToClipboard(copyString);
+    }
+
+    public async exportCellsToClipboard(
+        data: DbCellValue[][],
+        batchId: number,
+        resultId: number,
+        selection: ISlickRange[],
+        headersFlag,
+    ) {
+        let copyString = "";
+        if (headersFlag) {
+            copyString = this.addHeadersToCopyString(
+                copyString,
+                batchId,
+                resultId,
+                selection,
+            );
+        }
+
+        // create a mapping of rows to selections
+        let rowIdToSelectionMap = new Map<number, ISlickRange[]>();
+        let rowIdToRowMap = new Map<number, DbCellValue[]>();
+
+        // create a mapping of the ranges to get promises
+        let tasks = selection.map((range) => {
```

Comment:
Looks like this code is largely the same as above; can you refactor to dedupe

Reply:
this is the only instance of this, are you perhaps looking at a stale version?